### PR TITLE
[temp] feat: add import taxonomy button

### PR DESCRIPTION
**Pull request by rpenido**

### CODE REVIEW

File: src/taxonomy/data/thunks.js [hunk 1]
```diff
@@ -1,15 +1,75 @@
...
+    });
+
+    document.body.appendChild(fileInput);
+    fileInput.click();
+  });
+
+  const getTaxonomyName = () => {
+    let taxonomyName = null;
+    while (!taxonomyName) {
+      taxonomyName = prompt('Enter a name for the new taxonomy');
```
Comment (pomegranited) [inline src/taxonomy/data/thunks.js]: Strings should be translatable.

Comment (pomegranited) [reply src/taxonomy/data/thunks.js]: And using `prompt` and `alert` is temporary -- please add a comment at the top of this function to this effect so upstream know why we're doing it this way for now.

**Review by pomegranited — CHANGES_REQUESTED** [review]
@rpenido Could you have a look at [ADR 002](https://github.com/openedx/frontend-template-application/blob/master/docs/decisions/0002-feature-based-application-organization.rst)? According to that, we should add a new `import-tags` folder for this feature, with the components and functions in separate files underneath.

**Review by pomegranited — APPROVED** [review]
👍 This looks and works great @rpenido !

I'm not sure if we need tests added though? It might be difficult to test this temporary `prompt`/`alert` flow.

- [x] I tested this on my devstack with this branch merged with https://github.com/openedx/frontend-app-course-authoring/pull/655 so I could see the imported tags.
- [x] I read through the code
- [x] I checked for accessibility issues by using my keyboard to navigate.
- [x] Includes documentation
- [x] All strings are translatable.

**Comment by rpenido** [comment]
> I'm not sure if we need tests added though? It might be difficult to test this temporary prompt/alert flow.

I agree. We expect this code to be temporary. I will check if the coverage report would allow us to let it pass.

Closed in favor of:
- https://github.com/openedx/frontend-app-course-authoring/pull/675

**Pull request closed without merging**